spring-boot-project/spring-boot-docker-compose/src/test/java/org/springframework/boot/docker/compose/service/connection/postgres/PostgresEnvironmentTests.java

```diff
@@ -105,9 +106,9 @@ void getDatabaseWhenNoPostgresDbAndPostgresUser() {
 	}
 
 	@Test
-	void getDatabaseWhenNoPostgresqlDbAndPostgresUser() {
+	void getDatabaseWhenNoPostgresqlDatabaseAndPostgresqlUsername() {
 		PostgresEnvironment environment = new PostgresEnvironment(
-				Map.of("POSTGRESQL_USER", "me", "POSTGRESQL_PASSWORD", "secret"));
+				Map.of("POSTGRESQL_USERNAME", "me", "POSTGRESQL_PASSWORD", "secret"));
```

Comment:
This test should be added alongside `getDatabaseWhenNoPostgresqlDbAndPostgresUser`.